src/lib/features/feature-search/feature-search-controller.ts

```diff
@@ -71,13 +72,17 @@ export default class FeatureSearchController extends Controller {
         res: Response,
     ): Promise<void> {
         if (this.config.flagResolver.isEnabled('featureSearchAPI')) {
-            const { query, projectId, type } = req.query;
+            const { query, projectId, type, tag } = req.query;
             const userId = req.user.id;
+            const normalizedTag = tag
```

Comment:
Is this actually `tags` since it's an array?

Reply:
If we decide to make plural then I'll adjust all query params in one go. For now I prefer consistency